src/WebUI/ImageDetails/ImageDetails.tsx

```diff
@@ -125,6 +117,27 @@ export class ImageDetails extends BaseComponent<IImageDetailsProperties, IImageD
         );
     }
 
+    private _getImageDetailsRowsData(imageDetails: IImageDetails): ArrayItemProvider<any> {
+        let imageDetailsRows: any[] = [];
+        const digest: string = imageDetails.hash || "";
+        const imageType: string = imageDetails.imageType || "";
+        const mediaType: string = imageDetails.mediaType || "";
+        const registryName: string = this._getRegistryName();
+        const imageSize: string = ""; // ToDo: Size is currently not available in imageDetails.
+        const labels: string[] = imageDetails.tags || [];
+        const jobName: string = format("#{0} on {1}", imageDetails.buildId || "", imageDetails.buildDefinitionName || ""); // ToDo: JobName is currently not available in imageDetails.
```

Comment:
localization, also i thought this jobDetails suppose to be part of the header as per ppt.
![image](https://user-images.githubusercontent.com/26214977/55325415-a9b29900-54a2-11e9-9f3b-f713097085db.png)

Reply:
Shashank asked me to add the job text after labels. I will sync up with him on correct behavior